packages/astro/src/runtime/client/dev-overlay/overlay.ts

```diff
@@ -492,16 +492,20 @@ export class AstroDevOverlay extends HTMLElement {
 	setOverlayVisible(newStatus: boolean) {
 		const barContainer = this.shadowRoot.querySelector<HTMLDivElement>('#bar-container');
 		const devBar = this.shadowRoot.querySelector<HTMLDivElement>('#dev-bar');
+		const devBarHitboxAbove =
+			this.shadowRoot.querySelector<HTMLDivElement>('#dev-bar-hitbox-above');
 		if (newStatus === true) {
 			this.devOverlay?.removeAttribute('data-hidden');
 			barContainer?.removeAttribute('inert');
 			devBar?.removeAttribute('tabindex');
+			if (devBarHitboxAbove) devBarHitboxAbove.style.height = '0';
 			return;
 		}
 		if (newStatus === false) {
 			this.devOverlay?.setAttribute('data-hidden', '');
 			barContainer?.setAttribute('inert', '');
 			devBar?.setAttribute('tabindex', '0');
+			if (devBarHitboxAbove) devBarHitboxAbove.style.height = '42px';
```

Comment:
Could we move the `42` value to a const somewhere so we can change the hitbox height in one go if needed? Especially now that there's a calculated value from it (the -38)

Reply:
Yep that makes sense. I've updated things - more like that?